extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/util/AuthZUtils.scala

```diff
@@ -74,6 +85,10 @@ private[authz] object AuthZUtils {
   def getAuthzUgi(spark: SparkContext): UserGroupInformation = {
     // kyuubi.session.user is only used by kyuubi
     val user = spark.getLocalProperty("kyuubi.session.user")
+    if (StringUtils.isNotBlank(user)) {
```

Comment:
if `kyuubi.session.user.sign.enabled` is true and `kyuubi.session.user` is absent, we should fail w/ `AccessControlException`

Reply:
OK. If `kyuubi.session.user.sign.enabled` is enabled, force to check session user.